app/src/components/Avatar.css

```diff
@@ -0,0 +1,15 @@
+.img-s {
+  width: 120px;
+}
+
+.img-m {
+  width: 130px;
+}
+
+.img-l {
+  width: 140px;
+}
+
+.photoAvatar {
+  margin-bottom: 25px;
+}
```

Comment:
i dont like that `Avatar.jsx` class has this margin-bottom by default. It should be pure from side effects like this

Now each avatar you create will be with margin-bottom which is unexpexted cause by default we dont want it to be with margin

What I suggest:
1. Delete this `.photoAvatar` className
2. Create new prop in Avatar.jsx called `className` and add it to your className with other pre-defined class names. This will allow you to add specific class names for your avatar from outside depending on context. Basically you will be able to add different styles in different situations
3. In `Card.jsx` and `Card.css` add the code with your `margin-bottom` like here so it applied to `Avatar.jsx`